index.html

```diff
@@ -0,0 +1,442 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Sebastian M - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Sebastian M</h1>
```

Comment:
- [ ] Consider hiding the page title when the mobile menu is opened. 
- [ ] [OPTIONAL] Consider making the mobile header fixed so that the user can access it at any section of the page